Structure_Engine/Query/Description.cs

```diff
@@ -295,13 +299,156 @@ public static string Description(this LoadingPanelProperty property)
             return "LoadingPanel Application: " + property.LoadApplication + " RefEdge: " + property.ReferenceEdge;
         }
 
+        /***************************************************/
+        /**** Public Methods - Constraints              ****/
+        /***************************************************/
+
+        [Description("Generates a default description for the Constraint3DOF as 'DOftypes-Values'.")]
```

Comment:
DOFType values

Reply:
not sure on making this change. Intention was " 'DOF type description' - 'Any numerical values'"

Reply:
I think it's a bit too wordy, could we not make it simpler? 

Generates a default description for the Constraint3DOF based on the constraints at each degree of freedom.

Reply:
happy with that.